src/actions/sso/authenticationRequest.ts

```diff
@@ -4,33 +4,38 @@ import { Authentication } from 'jolocom-lib/js/interactionTokens/authentication'
 import { routeList } from 'src/routeList'
 import { ThunkAction } from '../../store'
 import { InteractionTransportType } from '@jolocom/sdk/js/src/lib/interactionManager/types'
-import { cancelSSO } from '.'
+import { cancelSSO, scheduleSuccessNotification } from '.'
 
 export const consumeAuthenticationRequest = (
   authenticationRequest: JSONWebToken<Authentication>,
-  channel: InteractionTransportType
+  channel: InteractionTransportType,
 ): ThunkAction => async (dispatch, getState, backendMiddleware) => {
   const { interactionManager } = backendMiddleware
   const interaction = await interactionManager.start(
     channel,
-    authenticationRequest
+    authenticationRequest,
   )
 
   return dispatch(
     navigationActions.navigate({
       routeName: routeList.AuthenticationConsent,
       params: {
         interactionId: interaction.id,
-        interactionSummary: interaction.getSummary()
+        interactionSummary: interaction.getSummary(),
       },
       key: 'authenticationRequest',
     }),
   )
 }
 
 export const sendAuthenticationResponse = (
-  interactionId: string
+  interactionId: string,
 ): ThunkAction => async (dispatch, getState, backendMiddleware) => {
-  const interaction = backendMiddleware.interactionManager.getInteraction(interactionId)
-  return interaction.send(await interaction.createAuthenticationResponse()).then(() => dispatch(cancelSSO))
+  const interaction = backendMiddleware.interactionManager.getInteraction(
+    interactionId,
+  )
+  return interaction
+    .send(await interaction.createAuthenticationResponse())
+    .then(() => dispatch(scheduleSuccessNotification))
+    .then(() => dispatch(cancelSSO))
```

Comment:
In some interaction we come back to home in some not, is it intentional ?

Reply:
Changed so all will navigate to the `InteractionScreen`